tpm2/test/policy_test.go

```diff
@@ -981,3 +1006,160 @@ func TestPolicyAuthValue(t *testing.T) {
 	}
 
 }
+
+func TestPolicyDuplicatonSelect(t *testing.T) {
```

Comment:
nit: typo (should be `TestPolicyDuplicationSelect`)

More substantive : this test only checks that the Update function calculates the right hash. Maybe it should really be called `TestPolicyDuplicationSelectUpdate`?